grid_map_costmap_2d/CMakeLists.txt

```diff
@@ -32,13 +23,44 @@ set(dependencies
   tf2_geometry_msgs
 )
 
+add_library(${PROJECT_NAME}
```

Comment:
Do we need to mark this target as `INTERFACE`?

Reply:
Yes, good catch!